modules/experimental/pages/guided.adoc

```diff
@@ -0,0 +1,16 @@
+= Guided
+
+`Unit ability`
+
+{blank}[TODO: add introductory text.]
+
+---
+
+Where your unit—or weapon system—has the {guided} ability:
+
+* When shooting at a marked enemy detachment, the unit—or weapon system—has Range 60 cm and the {artillery} special ability.
+
+
+.Related information
+* xref:markerlights.adoc[]
+* xref:special-rules:artillery.adoc[]
```

Comment:
As far as I can recall, this new topic is rather unique in the way that it mentions weapon systems.

1. Is this really necessary?
2. If it is really necessary, then I feel like we ought to reference the weapon systems text (more fiddly than usual, because we need to use the anchor for that sub-heading — I've done this before, but I can't 100% remember the syntax or find a definitive answer right now, so might have to fiddle with it to make it work properly).

```suggestion
* xref:special-rules:artillery.adoc[]
* xref:special-rules:shooting-with-war-engines.adoc#_weapon_systems[Weapon systems]
```

Reply:
> But it might be cleaner to keep all the references to 'unit', and then add a note that for War Engine units it only applies to individual weapon systems.

It seems like an odd outlier when we've been content to use plain 'unit' for the likes of the _Artillery_ ability. But I suppose that in due course we may *also* need to clarify how _Artillery_ would work for a single weapon system as opposed to a whole unit.

To be pragmatic for now and save messing about with a note, especially given that this is an experimental ability, we could just go with the unusual 'unit/weapon system' duality that we've drafted here. We can leave these minor but awkward questions about consistency and wider implications to address in the future.

On that basis, I'd be happy to resolve this conversation with no further action required at this time — although I suppose that we might as well apply the suggestion I made above to insert the xref to the weapon systems text just in case anyone stumbles over this unusual mention. (I checked — the 'anchor' code is correct in that suggestion.)